page-factory-api/src/main/java/ru/sbtqa/tag/pagefactory/reflection/Reflection.java

```diff
@@ -80,6 +69,23 @@
      */
     boolean isRequiredElement(Field field, String title);
 
+    /**
+     * Find specified Object by title annotation among current page fields
+     * @deprecated will be removed in future versions
+     *          use {@link Find}
+     *
+     * @param page the page on which the method is executing
+     * @param title title of the element to search
+     * @param <T> supposed type of the field. if field cannot be cast into this type, it will fail
+     * @return Object found by corresponding title
+     * @throws ru.sbtqa.tag.pagefactory.exceptions.PageException if failed to
+     * find corresponding element or element type is set incorrectly
+     */
+    @Deprecated
+    default <T> T getElementByTitle(Page page, String title) throws PageException {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Do not forget to remove this deprecated code someday. ([squid:S1133](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1133))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=146&issues=AWg8K_V7UMkE2J58ZhTe&open=AWg8K_V7UMkE2J58ZhTe)

[ISSUE_KEY]: AWg8K_V7UMkE2J58ZhTe